doc/index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
```

Comment:
Hi, apologies I didn't catch this, but this PR ended up checking in some generated HTML into source code, it's noisy, derived data and doesn't belong here.

I've got a commit to clean this up and prevent it in the future: https://github.com/18F/identity-idp/pull/6052/commits/96031277e5209cdd5d72077e78f337181a5cb325